React/nested-event-handling.md

```diff
@@ -124,6 +128,43 @@ Now let's take a look at an updated version of the diagram that we created in ou
 
 Note that we're passing in a new `isPresent` prop here. How could we modify our `Student` component to reflect this information?
 
+#### Removing State From `<Student />`
+
+Part of the concept of 'lifting state' is removing any state that is no longer relevant to the child element. 
+
+**Question:** What pieces of state still need to keep inside of the `Student`?
+
+<details>
+  <summary>Answer</summary>
+
+  Actually, none! While we _could_ make a working app that uses a class based component in the `Student` and the `StudentCollection`, it will be cleaner to read and update in the future if we turn the classical version of `Student` back into a functional component!
+</details>
+
+Ultimately, we can refactor the code to look like this:
+
+```javascript
+  // Student.js
+  const Student = (props) => {
+  return (
+    <section>
+      <h3>Student Component</h3>
+      <h4 className={attendanceClass}>Name {props.fullName} </h4>
+      <p>Email: {props.email} </p>
```

Comment:
The ordering here feels a little weird, given that the next section talks about displaying attendance as though we haven't thought about it yet.

Also, `attendanceClass` is undefined on line 151.

Reply:
I can think of two ways to proceed:
- Here, make it a true refactor (remove the `attendanceClass` and the CSS)
- Here talk about "let's include some code to show whether the student is present", and below change the text to focus specifically on marking a student present.